packages/types/lib/plans/db.ts

```diff
@@ -0,0 +1,38 @@
+import type { Timestamps } from '../db';
+
+export interface DBPlan extends Timestamps {
+    id: number;
+    account_id: number;
+    name: string;
+    trial_start_at: Date | null;
+    trial_end_at: Date | null;
+    trial_extension_count: number;
+    trial_end_notified_at: Date | null;
+
+    /**
+     * Limit the number of connections with active scripts
+     * Set to null to remove limit
+     * @default 3
+     */
+    max_connection_with_scripts: number | null;
```

Comment:
I would be consistent in the naming and follow what's done with the trial features aka `FEATURE_ATTRIBUTE`
```
connection_with_scripts_max
environments_max
...
```

Reply:
👍🏻